### ISSUE TITLE
Translation (into german)

### ISSUE THREAD

**Issue by Theodin** [fransschreuder/libosmscout#17]
In which files do I find translatable strings?
(Sorry for being lazy and not looking myself, but you know your code better).

**Comment by fransschreuder** [comment]
You don't need to go into the code, just take the Dutch translation as a template
https://github.com/fransschreuder/OSMScout-ubuntu/blob/master/OSMScout/OSMScout/po/osmscout_nl.po

**Comment by fransschreuder** [comment]
Hello Theodin, are you still interested in creating a German translation?

**Comment by Theodin** [comment]
Yes. I have it almost  finished. I want to try it on my phone first and see if everything makes sense. Can I just rename the file to osmscout_de.po and copy it over?

**Comment by fransschreuder** [comment]
if you have the project checked out, you can just copy it, add it to the OSMScout/OSMScout/OSMScout.pro file and also to the .qrc file
Then run:
lupdate OSMScout.pro
lrelease OSMScout.pro
Now build the click package

**Comment by fransschreuder** [comment]
Hello Theodin, have you still thought about this translation?

**Comment by Theodin** [comment]
Havent found the time and try to get my translation over to my phone. Sorry.

**Comment by ksalver** [comment]
it's done - erledigt   ksalver/... osmscout_de.po
